client/src/services/lab7/RepairService.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import API from "../API";
+
+const endpoints = {
+  SUBMIT_REPAIR: "/lab7/repair/submit",
+  UPDATE_REPORT: "/lab7/repair/update",
+};
+
+const RepairService = {
+  submitRepair: (activity, repair) => {
+    return API.postWithBody(
+      process.env.REACT_APP_SERVER_URL + endpoints.SUBMIT_REPAIR,
```

Comment:
```suggestion
      '$(process.env.REACT_APP_SERVER_URL)$(endpoints.SUBMIT_REPAIR)',
```

Reply:
This applies to any of the services that are included within the functionality of this lab

Reply:
Updated